app/tools/android/make_apk.py

```diff
@@ -690,6 +691,13 @@ def MakeEmbeddedApk(options, app_info, app_dir, packaged_archs):
       sys.exit(13)
 
 
+@atexit.register
+def PrintDeprecationMessage():
+    print('\n\n\n   WARNING!!! This command is deprecated and will be removed soon.\n\n'
```

Comment:
Nitpick 2: I think it's better not to indent the message, but that's just my opinion.